### PR TITLE
add important firewall announcement

### PR DESCRIPTION
**Pull request by mbrown555**

### CODE REVIEW

File: README.aspnet.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Important: Client Firewall Rules Update to Microsoft Container Registry (MCR)
+
+To provide a consistent FQDNs, on **March 3, 2020** the data endpoint will be changing from *.cdn.mscr.io to *.data.microsoft.com.
```
Comment (MichaelSimons) [inline README.aspnet.md]: There is still a trailing space at the end of this line.

File: README.aspnet.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## Important: Client Firewall Rules Update to Microsoft Container Registry (MCR)
+
+To provide a consistent FQDNs, on **March 3, 2020** the data endpoint will be changing from *.cdn.mscr.io to *.data.microsoft.com. 
+
+For more info, see [MCR Client Firewall Rules](https://aka.ms/mcr/firewallrules).
+---------------------------------------------------------------------------------
```
Comment (MichaelSimons) [inline README.aspnet.md]: There should be a blank line before the next header.

Comment (mbrown555) [reply README.aspnet.md]: fixed

File: README.aspnet.md (hunk 1 above)
Comment (mbrown555) [reply README.aspnet.md]: fixed

**Pull request merged**